internal/rbd/replicationcontrollerserver.go

```diff
@@ -596,12 +615,6 @@ func (rs *ReplicationServer) ResyncVolume(ctx context.Context,
 		}
 	}
 
-	util.UsefulLog(
```

Comment:
can you please keep this one as it is and log the local details.

Reply:
sure